### PR TITLE
Migrate change

### PR DESCRIPTION
**Pull request by Drewbrad4**
Removed the reserved column from the Books model and added more User info to our User model.

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -3,6 +3,16 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
+<div class="field">
```
Comment (sturdynut) [inline app/views/devise/registrations/new.html.erb:6]: Looks like your spacing is off here...

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -21,6 +31,26 @@
     <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
   </div>
 
+<div class="field">
```
Comment (sturdynut) [inline app/views/devise/registrations/new.html.erb:34]: Here too...

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -21,6 +31,26 @@
...
+    <%= f.label :address %><br />
+    <%= f.text_field :address, autofocus: true, autocomplete: "Address" %>
```
Comment (sturdynut) [inline app/views/devise/registrations/new.html.erb:36]: Did you mean to use `placeholder` vs `autocomplete`?  The others below also have this.

I noticed that `autofocus` was turned on for every `text_field`, maybe `autofocus` should only be on the first field?

**Review by sturdynut — COMMENTED** [review]
Looking great!  A couple of questions on the registration form.

File: app/views/devise/registrations/new.html.erb (hunk 3 above)
Comment (Drewbrad4) [reply app/views/devise/registrations/new.html.erb]: Yes, you were totally right on both accounts. Thanks!

File: app/javascript/components/App.js [hunk 4]
```diff
@@ -53,6 +53,8 @@ export default class App extends React.Component {
       current_user
     } = this.props
 
+    console.log(current_user)
```
Comment (talk2MeGooseman) [inline app/javascript/components/App.js:56]: Dirty console log remains D:

**Review by talk2MeGooseman — COMMENTED** [review]
Small comment

**Review by talk2MeGooseman — APPROVED** [review]
LGTM

File: app/javascript/components/App.js (hunk 4 above)
Comment (Drewbrad4) [reply app/javascript/components/App.js:56]: Haha sorry, we added it back to test something and forgot again. Gone now!

**Comment by Drewbrad4** [comment]
We were wondering, we think we have the relationships for all our tables correct now, but wanted to double check with you. Because Book is still holding a foreign-key for the user who owns it, it still needs a belongs_to :user relationship right?

**Pull request merged**